src/metabolic_ninja/resources.py

```diff
@@ -41,6 +54,7 @@ def get(self, task_id):
             return {
                 'id': result.id,
                 'state': result.state,
+                'info': result.info,
```

Comment:
What does `info` contain when the result is not ready? Is this the same as the arguments the task was called with?